css/main.css

```diff
@@ -4,7 +4,9 @@
    Author's custom styles
    ========================================================================== */
 
-
+#image-galleries .btn {
```

Comment:
I would make the change to image-gallery instead of galleries since it is a singular gallery or archive.